internal/controller/suite_test.go

```diff
@@ -137,9 +137,9 @@ var _ = BeforeSuite(func() {
 	Expect(err).ToNot(HaveOccurred())
 
 	err = (&PolicyServerReconciler{
-		Client:     k8sManager.GetClient(),
```

Comment:
(opening thread randomly here)

I understand dropping the unit tests, but I'm still sad about it. Any possibility to amend them in this PR instead?
If not, we should open an issue to track it.